ReSharper.FSharp/src/FSharp.Psi.Features/src/Parsing/FSharpImplTreeBuilder.fs

```diff
@@ -648,10 +662,9 @@ type FSharpImplTreeBuilder(lexer, document, decls, lifetime, path, projectedOffs
         | [] -> ()
         | binding :: rest ->
 
-        x.ProcessTopLevelBinding(binding)
+        x.ProcessTopLevelBinding(binding, false)
         for binding in rest do
-            x.AdvanceToTokenOrPos(FSharpTokenType.AND, binding.StartPos)
-            x.ProcessTopLevelBinding(binding)
+            x.ProcessTopLevelBinding(binding, true)
```

Comment:
The `isSecond` parameter is wrongly `true` for third and subsequent bindings.

Reply:
yes it should be called isSecondary